examples/guides/messaging/messaging-with-sqs/messaging-with-sqs.md

```diff
@@ -0,0 +1,298 @@
+- [Amazon SQS Account](https://aws.amazon.com/sqs/)
+
+
+## Implementation
+> If you want to skip the basics and move directly to the [Testing](#testing) section, you can download the project from GitHub and skip the [Implementation](#implementation) instructions.
+
+Take a look at the code samples below to understand how to implement the scenario. 
+
+Following code creates a new queue in Amazon SQS with the configuration provided in a file.
+
+**create_notification_queue.bal**
+```ballerina
+import ballerina/config;
+import wso2/amazonsqs;
+
+function createNotificationQueue(string queueName) returns string|error {
+    // Amazon SQS client configuration
+    amazonsqs:Configuration configuration = {
+        accessKey: config:getAsString("ACCESS_KEY_ID"),
+        secretKey: config:getAsString("SECRET_ACCESS_KEY"),
+        region: config:getAsString("REGION"),
+        accountNumber: config:getAsString("ACCOUNT_NUMBER")
+    };
+
+    // Amazon SQS client
+    amazonsqs:Client sqsClient = new(configuration);
+
+    // Create SQS Standard Queue for notifications
+    string|error queueURL = sqsClient->createQueue(queueName, {});
+    return queueURL;
+}
+```
+
+Following code generates fire alert notifications periodically and are sent to the above created SQS queue.
+
+**notify_fire.bal**
+```ballerina
+import ballerina/config;
+import ballerina/log;
+import ballerina/runtime;
+import wso2/amazonsqs;
+
+// periodicallySendFireNotifications, which periodically send fire alerts to Amazon SQS queue
+function periodicallySendFireNotifications(string queueResourcePath) {
+
+    // Amazon SQS client configuration
+    amazonsqs:Configuration configuration = {
+        accessKey: config:getAsString("ACCESS_KEY_ID"),
+        secretKey: config:getAsString("SECRET_ACCESS_KEY"),
+        region: config:getAsString("REGION"),
+        accountNumber: config:getAsString("ACCOUNT_NUMBER")
+    };
+
+    // Amazon SQS client
+    amazonsqs:Client sqsClient = new(configuration);
+
+    while (true) {
+
+        // Wait for 5 seconds
+        runtime:sleep(5000);
+        string queueUrl = "";
+
+        // Send a fire notification to the queue
+        amazonsqs:OutboundMessage|error response = sqsClient->sendMessage("There is a fire!", 
+            queueResourcePath, {});
+        // When the response is valid
+        if (response is amazonsqs:OutboundMessage) {
+            log:printInfo("Sent an alert to the queue. MessageID: " + response.messageId);
+        } else {
+            log:printError("Error occurred while trying to send an alert to the SQS queue!");
+        }
+    }
+
+}
+```
+
+Following code listens to the SQS queue and if there are any notifications, it would receive from the queue and delete the existing messages in the queue.
+
+**listen_to_fire_alarm.bal**
```

Comment:
Shall we add this code segment using INCLUDE_CODE tag?
Please refer https://github.com/wso2/ballerina-integrator/tree/master/www#a-include-a-code-file

Reply:
Added INCLUDE_CODE tag and moved the code to correct guide location.